### DISCUSSION TITLE
Manual Support Painting not working

### DISCUSSION THREAD

**Discussion by chrini1** [OrcaSlicer/OrcaSlicer#4041] [Q&A]
When clicking on Support Drawing in the top menu, my object and the plate dissapear in the main section of the window. 

![image](https://github.com/SoftFever/OrcaSlicer/assets/19890987/eb2d6308-db77-44dc-a36d-b93dd7fbd35c)

Any idea why?

V1.9.0

**Comment by ViniSug** [comment]
Fixed on v1.9.1
